src/v2/Apps/Order/OrderApp.tsx

```diff
@@ -87,35 +130,41 @@ class OrderApp extends React.Component<OrderAppProps, {}> {
       )
     }
 
-    const stripePromise = loadStripe(sd.STRIPE_PUBLISHABLE_KEY);
+    const stripePromise = loadStripe(sd.STRIPE_PUBLISHABLE_KEY)
     return (
       <SystemContextConsumer>
         {({ isEigen, mediator }) => {
           this.mediator = mediator
           return (
-            <MinimalNavBar to={artworkHref}>
-              <AppContainer>
-                <Title>Checkout | Artsy</Title>
-                {isEigen ? (
-                  <Meta
-                    name="viewport"
-                    content="width=device-width, user-scalable=no"
-                  />
-                ) : (
-                  <Meta
-                    name="viewport"
-                    content="width=device-width, initial-scale=1, maximum-scale=5 viewport-fit=cover"
-                  />
-                )}
-                <SafeAreaContainer>
-                  <Elements stripe={stripePromise}>
-                    <>{children}</>
-                  </Elements>
-                </SafeAreaContainer>
-                <StickyFooter orderType={order.mode} artworkId={artworkId} />
-                <ConnectedModalDialog />
-              </AppContainer>
-            </MinimalNavBar>
+            <Box>
+              {/* FIXME: remove once we refactor out legacy backbone code.
+                  Add place to attach legacy flash message, used in legacy inquiry flow
+              */}
+              <div id="main-layout-flash" />
```

Comment:
Note: Placing `<div id="main-layout-flash" />` inside of `MinimalNavBar` prevents the flash message from being shown.